src/test/resources/corrupted/build-info.json

```diff
@@ -0,0 +1,7 @@
+{
```

Comment:
I used the data from https://beats-ci.elastic.co/blue/organizations/jenkins/e2e-tests%2Fe2e-testing-mbp/detail/7.11.x/8/artifacts